lib/levels.js

```diff
@@ -55,7 +55,8 @@ function genLsCache (instance) {
   const formatter = instance[formattersSym].level
   const { labels } = instance.levels
   const cache = {}
-  for (const label in labels) {
+  /* eslint no-var: off */
+  for (var label in labels) {
```

Comment:
It's `for (let x = 0; x < 10; x += 1)` that is slower. I believe in this case we can do:

```js
for (const label in labels)
```

Correct, @mcollina ?

Reply:
Yes, correct.

Reply:
So everywhere we have a `for...in` in this code, please remove the eslint comment and adjust to a `for(const .. in ..)`.

Reply:
Done